README.md

```diff
@@ -37,7 +36,7 @@ We also do a weekly call on Monday at [14:00 UTC](https://www.worldtimebuddy.com
 
 Alby supports
 
-- All [Chromium based browsers](https://en.wikipedia.org/wiki/Chromium_(web_browser)#Browsers_based_on_Chromium) - Chrome Opera, Brave etc.
+- All [Chromium based browsers](<https://en.wikipedia.org/wiki/Chromium_(web_browser)#Browsers_based_on_Chromium>) - Chrome Opera, Brave etc.
```

Comment:
what's this link syntax actually?

Reply:
Prettier adjusted it. I assuem it's because the link itself already is using `)` which might be confusing.